perun-core/src/main/java/cz/metacentrum/perun/core/entry/ServicesManagerEntry.java

```diff
@@ -100,7 +100,10 @@ public void blockServiceOnDestination(PerunSession sess, Service service, int de
 	@Override
 	public void blockServicesOnDestinations(PerunSession sess, List<RichDestination> richDestinations) throws PrivilegeException, DestinationNotExistsException, ServiceAlreadyBannedException, FacilityNotExistsException {
 		for (RichDestination richDestination : richDestinations) {
-			blockServiceOnDestination(sess, richDestination.getService(), richDestination.getId());
```

Comment:
I think this should be considered as breaking change, since it changes expected outcome of the API call. You should update interface class for this method and also `ServicesManagerMethod.java` and RPC API docs accordingly as `ServiceAlreadyBannedException` is no longer thrown.